both/collections/quizzes.js

```diff
@@ -0,0 +1,35 @@
+Quizzes = new Mongo.Collection('quizzes');
+
+QuizzesSchema.AnswerOptionSchema = new SimpleSchema({
```

Comment:
This may be redundant, as individual options may be marked as `correctAnswer` directly.